test/controllers/hosts_controller_test.rb

```diff
@@ -39,12 +39,13 @@ def host_attributes(host)
   end
 
   test "should get csv index with data" do
-    host = FactoryBot.create(:host, :with_hostgroup, :on_compute_resource, :with_reports)
+    User.current.table_preferences.create(name: 'hosts', columns: ['name', 'os_title', 'owner', 'hostgroup', 'last_report'])
+    host = FactoryBot.create(:host, :with_hostgroup, :with_reports)
     get :index, params: { :format => 'csv', :search => "name = #{host.name}" }, session: set_session_user
     assert_response :success
     buf = response.stream.instance_variable_get(:@buf)
-    assert_equal "Name,Operatingsystem,Compute Resource Or Model,Hostgroup,Last Report\n", buf.next
```

Comment:
Just to not break previous CSV reports, I guess it's better to leave and test this entry as well.